unity/Assets/Scripts/ArtGallery/Controller/ArtGalleryController.cs

```diff
@@ -58,6 +58,11 @@ void Start()
         // Update is called once per frame
         void Update()
         {
+            if (Input.GetKeyUp(KeyCode.PageUp))
+            {
+                AdvanceLevel();
+            }
+
```

Comment:
Yes, some of the later levels seem to fail more often, so that's why it was added. It's not necessary to be merged, so I'll remove it before the merge.